### ISSUE TITLE
What hidden test are in the OOP lab section

### ISSUE THREAD

**Issue by babdulhakim2** [lethalskillzz/Andela-Proctor-Labs#1]
The section where you create super class of bank accounts with two sub classes. All visible test are ok, but what about the hidden test there

**Comment by Oladotun-Fasogbon** [comment]
Hello,

It's seems you mistakenly sent me a message, by the way I am preparing for Andela home-study test and wouldn't mind if you could be of help. Thanks.

**Comment by lethalskillzz** [comment]
They where passing as well

**Comment by lethalskillzz** [comment]
it wasn't a mistake you're watching this repo therefore will be notified regarding any new development about it.

**Comment by babdulhakim2** [comment]
Ok, just to clarify, do you have to pass 100% of the proctor to get qualified ?

**Comment by lethalskillzz** [comment]
i don't how they asses the score but it's unlikely that you to pass 100%.

**Comment by ucinho** [comment]
could someone help me in my andela labs ?

**Comment by ucinho** [comment]
plssssss

**Comment by uniquedynamo** [comment]
Good day all. Pleaseany help on Andela Fellowship cycle xviii home study assessment test; the test labs

**Comment by ucinho** [comment]
Are you in lagos ? Maybe I can help

**Comment by uniquedynamo** [comment]
Yes!! I'm in Lagos. I've not found it easier to complete the python home
tests. Thanks sir.

On 24 Nov 2016 12:37, "Uvinci Chris" <notifications@github.com> wrote:

Are you in lagos ? Maybe I can help

—
You are receiving this because you commented.
Reply to this email directly, view it on GitHub
<https://github.com/lethalskillzz/Andela-Proctor-Labs/issues/1#issuecomment-262756285>,
or mute the thread

**Comment by ucinho** [comment]
Which part of lagos are you ? I am currently at lekki. Maybe we can plan a meet up or as well, give me a call on (zero eight one thirty fortyfour seven one two seven) text me before calling

**Comment by uniquedynamo** [comment]
I'm actually on the mainland. IyanaIpaja. Okay. I'll be happy to get some
help from you.

**Comment by esirK** [comment]
hey Guys Am Currently getting this error /bin/sh: 1: python/nose2/bin/nose2: not found. Anyone tell me what it is n how to solve it??

**Comment by walemen52** [comment]
Im currently preparing as well, any one who can advice/help me to be on the right track?
I'm in Lagos.

**Comment by Dennis-612** [comment]
Can someone please help me on Andela lab tests
